Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -558,23 +526,37 @@ def get_attributes_command(client: Client, args: Dict[str, str], params: Dict[st
     )
 
 
-def fetch_attributes_command(client: Client, params: Dict[str, str]) -> List[Dict]:
+def fetch_attributes_command(client: Client, params: Dict[str, str]):
     """
-    Wrapper for fetching indicators from the feed to the Indicators tab.
+    Fetching indicators from the feed to the Indicators tab.
     Args:
         client: Client object with request
         params: demisto.params()
     Returns: List of indicators.
+
     """
     tlp_color = params.get('tlp_color')
     reputation = params.get('feedReputation')
     tags = argToList(params.get('attribute_tags', ''))
     feed_tags = argToList(params.get("feedTags", []))
     attribute_types = argToList(params.get('attribute_types', ''))
     query = params.get('query', None)
-    indicators = fetch_indicators(client, tags, attribute_types, query, tlp_color,
-                                  params.get('url'), reputation, feed_tags)
-    return indicators
+    params_dict = clean_user_query(query) if query else build_params_dict(tags=tags, attribute_type=attribute_types, limit=2000,
+                                                                          page=1)
+    search_query_per_page = client.search_query(params_dict)
+    while len(search_query_per_page.get("response", {}).get("Attribute", [])):
+        demisto.debug(f'search_query_per_page number of attributes:\
+                      {len(search_query_per_page.get("response", {}).get("Attribute", []))}\
+                        page: {params_dict["page"]}')
+        indicators = build_indicators(search_query_per_page, attribute_types, tlp_color, params.get('url'), reputation, feed_tags)
+        demisto.createIndicators(indicators)
+        params_dict['page'] += 1
+        search_query_per_page = client.search_query(params_dict)
+    if error_message := search_query_per_page.get('Error'):
+        raise DemistoException(f"Error in API call - check the input parameters and the API Key. Error: {error_message}")
+    params_dict.pop("limit", None)
+    params_dict.pop("page", None)
+    demisto.setLastRun({'params': params_dict})
```

Comment:
Do we use the last run anywhere in the code? If not, lets remove it

Reply:
ok, removing